frontend/src/components/game/GameNotificationContainer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GameNotification, NotificationType } from '../../api/GameNotification';
+
+const GameNotificationBox = styled.div`
+  position: absolute;
+  bottom: 25px;
+  left: 25px;
+  padding: 25px;
+  max-width: 20vw;
+  font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  text-align: center;
+  color: ${({ theme }) => theme.colors.gray};
+  background: ${({ theme }) => theme.colors.white};
+  box-shadow: ${({ theme }) => theme.colors.lightGray} 0px 8px 24px;
+  border-radius: 5px;
+  z-index: 1;
+
+  &:hover {
+    background: ${({ theme }) => theme.colors.background};
+    transition: 0.5s all;
+  }
+`;
+
+const notificationToString = (notification: GameNotification): string => {
+  switch (notification.notificationType) {
+    case NotificationType.SubmitCorrect: {
+      return `${notification.initiator.nickname} just submitted correctly
+        ${notification.content ? `, taking ${notification.content} place, ` : ''}.`;
+    }
```

Comment:
For sure, I think that's fair! I also think there's an argument for some of the simpler notifications to be frontend-only, as that is a simpler and faster technical solution, but it makes sense not to have too much logic in the frontend. Is it OK if that's in a future PR rather than this one @alankbi?

Reply:
Yep that's what I was thinking too